viper.go

```diff
@@ -1778,6 +1792,14 @@ func (v *Viper) SetConfigType(in string) {
 	}
 }
 
+// SetKeyCaseSensitivity disables the default behaviour of
+// case-insensitivising keys and preserves the key casing
+// as they are found in the config files.
+func SetKeyCaseSensitivity(on bool) { v.SetKeyCaseSensitivity(on) }
```

Comment:
Again, case sensitivity is not boolean IMHO. I would rather call this something like `SetKeysCaseSensitive` or `CaseSensitiveKeys`.